bin/mk_image

```diff
@@ -241,7 +241,7 @@ sub fix_alternatives_link
     unlink "$dir/usr/sbin/alternatives";
     unlink "$dir/var/log/update-alternatives.log";
     system "rm -r $dir/etc/alternatives/";
-    system "rm -r $dir/var/lib/rpm/alternatives/";
+    system "rm -r $dir/var/lib{/rpm,}/alternatives/";
```

Comment:
One of the paths will be missing and you'll always get an error. `rm -rf` would be better.

Reply:
I was debating with myself about it - the current one is reporting an error in the log and it did not harm, which is why I did leave it at that...

But I agree - no warning is cleaner; will change to -rf